third-party/axe/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "license": "MPL-2.0",
+  "//": "Any directory that uses `import ... from` or `export ...` must be type module. Temporary file until root package.json is type: module"
```

Comment:
There is no `"type": "module"` anymore, should this comment refer to the license or can we remove this file?